src/pms/cli.py

```diff
@@ -25,6 +27,9 @@
     main.command(name=ep.name)(ep.load())
 
 
+logging.basicConfig(level=os.getenv("LEVEL", "WARNING"))
```

Comment:
maybe this should be moved to `callback` and only apply to `pms.logger` instead of the root`logger`
What do you think?

Reply:
smoothing like

``` diff
@main.callback()
def callback(
...
-    if debug:  # pragma: no cover
-        logger.setLevel("DEBUG")
+    logger.setLevel("DEBUG" if debug else os.getenv("LEVEL", "WARNING"))
```

Reply:
Works for me 👍. 

Only potential caveat might be if one of the dependencies has its own logger - then its logs would appear in the CLI output. I think most dependencies will be silent, but perhaps one like InfluxDB might emit some logs without this. Do you recall any logs from other dependencies when using the CLI?

For now, I've amended the first commit and force pushed over the top.

Happy to go back again if you decide root logger is better after all.

Reply:
> Do you recall any logs from other dependencies when using the CLI?

The MQTT library puts quite a lot of debug log messages, but I do not think that those are needed hare.
The `--debug` option is intended for debug messages from this library.